.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
We may place here links the same as here - https://github.com/microsoft/azure-pipelines-agent/blob/1096d5d8fff01a7ac571d4e91f339b3a1a3f63ec/.github/ISSUE_TEMPLATE/config.yml